### PR TITLE
feat: Add a function to determine the accountId of an LBP pool

### PR DESCRIPTION
**Pull request by vaclav-slavik**
Implements #77

### CODE REVIEW

**Comment by mckrava** [comment]
Storybook testing artifacts:
- [storybook-ci-tests-results](https://github.com/galacticcouncil/Basilisk-ui/suites/5398893612/artifacts/170210746);
Or check GH Action `Storybook testing CI flow` for more logs.

**Review by dexterslabor — APPROVED** [review]

File: src/hooks/pools/lbp/rpc/decorators.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { getPoolAccount as getLpbPollAccount } from './getPoolAccount';
+
+export const lbp = {
+  getLpbPollAccount,
```
Comment (dexterslabor) [inline src/hooks/pools/lbp/rpc/decorators.tsx]: This does not work for 2 reasons:

1. the typo for `pool`
2. the decorator needs to have the same name as in the RPC method, which is `getPoolAccount`

File: src/hooks/polkadotJs/usePolkadotJs.tsx [hunk 2]
```diff
@@ -10,29 +10,7 @@ import { useUnmount } from 'react-use';
...
-};
-const rpc = {
-  xyk: {
-    getPoolAccount,
-  },
-  lbp: {
-    getPoolAccount,
-  },
-};
+import { rpc } from '../pools/rpcDecorators';
```
Comment (dexterslabor) [inline src/hooks/polkadotJs/usePolkadotJs.tsx:13]: I do not believe this was tested, because it does not work.

**Review by dexterslabor — CHANGES_REQUESTED** [review]
Implementation does not work. See comments

**Review by dexterslabor — APPROVED** [review]

**Comment by mckrava** [comment]
Storybook testing artifacts:
- [storybook-ci-tests-results](https://github.com/galacticcouncil/Basilisk-ui/suites/5539618175/artifacts/178240707);
Or check GH Action `Storybook testing CI flow` for more logs.

**Comment by mckrava** [comment]
<h3>Barecheck - Code coverage report</h3>Total: <b>5.75%</b>

Your code coverage diff: <b>-0.01% ▾</b>

<details><summary>Uncovered files and lines</summary><table><tbody><tr><th>File</th><th>Lines</th></tr><tr><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx">src/hooks/polkadotJs/usePolkadotJs.tsx</a></td><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L15">15</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L20">20</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L30">30</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L33-L34">33-34</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L37-L39">37-39</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L41-L43">41-43</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L49-L50">49-50</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L56">56</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L60-L61">60-61</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L66-L68">66-68</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L76-L79">76-79</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L85">85</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L88-L90">88-90</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L94-L96">94-96</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L101-L104">101-104</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L108">108</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L119-L121">119-121</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L123">123</a>, <a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/polkadotJs/usePolkadotJs.tsx#L126">126</a></td></tr><tr><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/pools/rpcDecorators.tsx">src/hooks/pools/rpcDecorators.tsx</a></td><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/pools/rpcDecorators.tsx#L3">3</a></td></tr><tr><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/pools/lbp/rpc/decorators.tsx">src/hooks/pools/lbp/rpc/decorators.tsx</a></td><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/pools/lbp/rpc/decorators.tsx#L4">4</a></td></tr><tr><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/pools/lbp/rpc/getPoolAccount.tsx">src/hooks/pools/lbp/rpc/getPoolAccount.tsx</a></td><td><a href="https://github.com/galacticcouncil/Basilisk-ui/blob/bc9bcfc78424dacab89b9e509550433224c8b353/src/hooks/pools/lbp/rpc/getPoolAccount.tsx#L7">7</a></td></tr></tbody></table></details>

**Pull request closed without merging**